post-office-stub/src/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "F2F",
+  "version": "1.0.0",
+  "description": "OneLogin GovNotify stub",
+  "repository": "https://github.com/alphagov/di-ipv-cri-f2f-api",
+  "author": "GDS",
+  "license": "MIT",
+  "dependencies": {
+    "@aws-lambda-powertools/logger": "1.5.1",
+    "@aws-lambda-powertools/metrics": "1.5.1",
+    "@aws-sdk/credential-providers": "3.245.0",
+    "aws-xray-sdk-core": "^3.4.1",
+    "esbuild": "0.14.14"
+  },
+  "devDependencies": {
+    "@aws-cdk/assertions": "^1.172.0",
+    "@aws-cdk/yaml-cfn": "^1.172.0",
+    "@types/aws-lambda": "^8.10.109",
+    "@types/jest": "^29.4.0",
+    "@types/js-yaml": "^4.0.5",
+    "@types/node": "^18.11.18",
+    "@types/node-jose": "^1.1.10",
+    "@typescript-eslint/eslint-plugin": "^5.50.0",
+    "@typescript-eslint/eslint-plugin-tslint": "^5.50.0",
+    "eslint": "^8.32.0",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jest": "^27.2.1",
+    "eslint-plugin-jsdoc": "^39.6.8",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-security": "^1.6.0",
+    "prettier": "^2.8.4",
+    "ts-jest": "^29.0.5",
```

Comment:
Is there plans to add unit tests? If not could we remove the jest and jest related dependancies to reduce the amount of dependabot PRs we get